dbld/builddeps

```diff
@@ -43,6 +43,7 @@ function install_dbld_dependencies()
             fi
             ;;
         fedora)
+            $DNF_INSTALL -y wget dnf-plugins-core
```

Comment:
but shouldn't we remove them from add_copr_repo() at the same time?

Reply:
Nope, `dnf builddep` is from `dnf-plugins-core`, if we would remove these packages then `install_rpm_build_deps` wouldn't work.

I guess `virtualenv` is the answer, or at least install packages per-user  and not system-wide.